docs/source/prereqs/2-architecture.rst

```diff
@@ -1,10 +1,10 @@
 .. _hq-architecture:
 
-Architecture Overview
+Architecture and Platform Overview
 =====================
 
 It is useful to have some context on what it is that you are hosting,
 when you are hosting CommCare HQ.
 
 A good place to find that context is the `Architecture Overview <https://commcare-hq.readthedocs.io/overview/architecture.html>`_
-in the CommCare HQ documentation.
+and the `Platform Overview <https://commcare-hq.readthedocs.io/overview/platform.html>`_ the CommCare HQ documentation.
```

Comment:
```suggestion
and the `Platform Overview <https://commcare-hq.readthedocs.io/overview/platform.html>`_ in the CommCare HQ documentation.
```

Reply:
See [this](https://github.com/dimagi/commcare-cloud/pull/6239/commits/12080bc961e3f12025ec8c1b1fc4e2b3690bbe8a).